domains/reclamations-v1.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "SoulFlance95",
+      "email": "sedare2017@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://reclamations-app.vercel.app/"
```

Comment:
Invalid CNAME, CNAME records should be just hostnames with no file paths or protocols.